blocks/media-upload-button/index.js

```diff
@@ -78,11 +92,19 @@ class MediaUploadButton extends Component {
 		}
 
 		if ( gallery ) {
```

Comment:
I see what you're getting at. Yeah, we could probably use a similar method in both cases.